airflow/models/dagrun.py

```diff
@@ -1679,6 +1679,10 @@ def get_log_filename_template(self, *, session: Session = NEW_SESSION) -> str:
         return self.get_log_template(session=session).filename
 
 
+def _dagrun_get_partial_task_ids(dag: DAG) -> list[str] | None:
```

Comment:
nit: why the prefix `dagrun_`?

Reply:
seemed to generic. Yeah. It's in the module, so I can remove it.

Reply:
Should be better now